### DISCUSSION TITLE
What is the name of the famous football stadium in Rio de Janeiro, the largest in Brazil?

### DISCUSSION THREAD

**Discussion by ParanoidUser** [ParanoidUser/yolo#617] [Galaxy Trivia] [answered]
**Bonus:** What is its capacity?

**Comment by peterbaccaro** [comment]
The famous football stadium in Rio de Janeiro, Brazil is called the **Maracanã Stadium**, or Estádio do Maracanã in Portuguese. It is also known as the Maracanã or Maraca. The Maracanã Stadium is one of the largest football stadiums in the world and has a rich history, having hosted numerous iconic football matches, including the final match of the 1950 FIFA World Cup. It is a prominent landmark in Rio de Janeiro and a symbol of Brazilian football culture.

**Comment by MohammedNayeem9** [comment]
The **Maracana** in Rio de Janeiro is the largest stadium in Brazil, with **78,838** seats(capacity).

**Comment by thuongtruong1009** [comment, marked as answer]
The name of the famous football stadium in Rio de Janeiro, Brazil is Maracanã Stadium (Estádio do Maracanã in Portuguese). As of my knowledge cutoff in 2021, its capacity was 78,838 spectators. However, it's worth noting that the stadium has undergone renovations and upgrades in recent years, so the current capacity may differ slightly from this number.
